server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -251,7 +251,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.deletion.docs.current",
+                "es.node.stats.indices.deletion.docs.total",
```

Comment:
Same

Reply:
I am not sure, the javadoc for   `getIndexFailedCount`  says `The number of failed indexing operations`
both `total` and `count` feel ok to me..
maybe it should not be a longGauge but asyncCounter